platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationImpl.java

```diff
@@ -88,8 +88,7 @@ public ApplicationImpl(Repository repo) {
       version = mainFeature.getVersion();
       description = mainFeature.getDescription();
     } else {
-      Optional<Feature> first =
-          autoInstallFeatures.stream().filter(f -> name.equals(f.getName())).findFirst();
+      Optional<Feature> first = features.stream().filter(f -> name.equals(f.getName())).findFirst();
```

Comment:
@jhunzik, could you add a new unit test for checking the main feature for a repo just like https://github.com/codice/ddf/blob/12e0418931df4666dfcde7414b5b561d9a1466ad/platform/admin/core/admin-core-appservice/src/test/resources/test-features-with-main-feature.xml#L33-L32 but with no auto install apps and just one feature with the same name as the repo?